packages/commercetools/theme/pages/Checkout/Payment.vue

```diff
@@ -131,7 +131,7 @@
           :selected="chosenPaymentMethod.value"
```

Comment:
if you are committing something to the `next` branch please don't modify theme (just don't commit these changes, keep them locally)